client/components/tinymce/plugins/wpcom-view/plugin.js

```diff
@@ -854,23 +856,32 @@ function wpview( editor ) {
 
 	editor.addButton( 'wp_view_remove', {
 		tooltip: i18n.translate( 'Remove' ),
-		icon: 'dashicon dashicons-no',
+		icon: 'dashicon dashicons-trash',
 		onClick: function() {
 			selected && removeView( selected );
 		}
 	} );
 
 	editor.once( 'preinit', function() {
-		toolbar = editor.wp._createToolbar( [
+		editToolbar = editor.wp._createToolbar( [
 			'wp_view_edit',
 			'wp_view_remove'
 		] );
+		removeToolbar = editor.wp._createToolbar( [
+			'wp_view_remove'
+		] );
 	} );
 
 	editor.on( 'wptoolbar', function( event ) {
 		if ( selected ) {
+
+			// Checks for ids in the gallery shortcode
+			const { attrs: { named: { ids } } } = Shortcode.parse(
```

Comment:
`parse` is not guaranteed to assign `attrs` to the return value:

https://github.com/Automattic/wp-calypso/blob/96035ca/client/lib/shortcode/index.js#L152-L154

When post content includes a `[gallery]` shortcode without any attributes, I'm seeing an error. It appears this is also a problem in `GalleryShortcode` ([reference](https://github.com/Automattic/wp-calypso/blob/96035caaf08ff68e7e2ffec3b597e267da329503/client/components/gallery-shortcode/index.jsx#L90)).